packages/components/tag/src/tag.vue

```diff
@@ -1,27 +1,27 @@
 <template>
   <span
     v-if="disableTransitions"
-    :class="classes"
+    :class="containerKls"
     :style="{ backgroundColor: color }"
     @click="handleClick"
   >
-    <span :class="ns.e('content')">
+    <span :class="contentKls">
       <slot />
     </span>
-    <el-icon v-if="closable" :class="ns.e('close')" @click.stop="handleClose">
+    <el-icon v-if="closable" :class="iconKls" @click.stop="handleClose">
```

Comment:
There is only one style that does not need to be transformed with a computed property.

Reply:
updated